docs/details/api.md

```diff
@@ -1,4 +1,4 @@
-# Exploring PMM API
+# Swagger API
```

Comment:
I would try to keep this without "swagger" because this is just a way how we let the user to Explore our API  (yes.. we embedded it in the URL already( ) 
We might replace swagger with something else later

Reply:
Yep, I had that change on my list for after release.